### DISCUSSION TITLE
Is there a way to disable some plugins in certain folders?

### DISCUSSION THREAD

**Discussion by einenlum** [starship/starship#6280] [Q&A]
It seems like the `detect_folders` key is to provide a list of folders to detect to enable the plugin.
My question is: is there a way to disable a plugin when we are *in* a specific folder?

Python keeps popping in my home directory because I have a python file somewhere.

**Comment by jankatins** [comment]
Wait for https://github.com/starship/starship/pull/4439 and then use direnv to only enable in certain dirs.
